New Terms: Nmap no ping, no port (#1300)
Codecademy/docs: content/cybersecurity/concepts/nmap/terms/no-ping-scan/no-ping-scan.md

---
Title: 'No Ping Scan'
Description: 'A Nmap no ping scan prevents Nmap from doing any host discovery.'
Subjects:
  - 'Computer Science'
  - 'Networking'
Tags:
  - 'Networking'
  - 'Cybersecurity'
  - 'Ethical Hacking'
CatalogContent:
  - 'introduction-to-cybersecurity'
  - 'paths/scan-systems-with-nmap'
---

> A Nmap **no ping scan** prevents Nmap from doing any host discovery. It is specified with the -Pn option. By default, Nmap only probes active machines found via host discovery. With this option selected, Nmap will do the requested scanning functions against every target IP specified, as if every one is active. This is done when it is important to find every possible active machine, including ones that may not be responsive to host discovery.

Reviewer: ```suggestion
A **no ping scan** prevents Nmap from doing any host discovery. It is specified with the `-Pn` option. By default, Nmap only probes active machines found via host discovery. With this option selected, Nmap will do the requested scanning functions against every target IP specified, as if every one is active. This is done when it is important to find every possible active machine, including ones that may not be responsive to host discovery.
```

Would it be worth unpacking what host discovery means here? Or should this be capture in a separate concept entry?

Author: @Dusch4593 I think it may be worth its own entry, since it's applicable to a number of other entries.

> **Note:** This option is often combined with a [no port scan (-sn)](https://www.codecademy.com/resources/docs/cybersecurity/nmap/no-port-scan).

## Syntax

```pseudo
nmap -Pn <target>
```

> This performs a scan of <target> without doing host discovery.

Reviewer: ```suggestion
This performs a scan of `<target>` without doing host discovery.
```

## Example

> The following shows the output of `nmap -Pn scanme.nmap.org`:

Reviewer 2: ```suggestion
The following example runs a no ping scan  on the site [scanme.nmap.org](http://scanme.nmap.org/):

```shell
nmap -Pn scanme.nmap.org
```
```
